### ISSUE TITLE
Number of channel-setup you would recommend?

### ISSUE THREAD

**Issue by 81N55E** [Mensen/ept_TFCE-matlab#10]
Hi Armand,

I am planning to use your toolbox for a spectral power analysis in an EEG sleep study. 
However I only have a 64 channel EEG setup? Would you still recommend doing a TFCE?
Just out of curiosity - what would be the lowest number of EEG channels to still have a reasonable argument why using this analysis? 
Thanks & best,
Mitja

**Comment by Mensen** [comment]
Hey Mitja! How are you?

I would say that anytime you have a unique neighbour structure between channels, its worthwhile to take the spatial information into account. For example, with just 4 channels, each channel will neighbour all the others, and so the spatial cluster will always be the same size and basically you are just scaling your data and the stats will be equal. On the other hand, with 6 channels, (e.g. polysomnography), it should help to know that the frontal electrodes are closer to the central ones than the posterior ones and the spatial clustering approach within TFCE will use that information. For example, if you were to find (without spatial clustering), that the frontal left, and the occipital right showed similar differences at p = 0.04, or that the frontal left and frontal right also did at p = 0.04 ... I'm much more likely to believe the latter is actually true.

So in short, your 64 channel recordings would definitely benefit from the approach!

I thought I replied to this yesterday already, but since it hasn't appeared, I assume I forgot to present "comment"... apologies if you get the same answer twice though.

**Comment by 81N55E** [comment]
Great, Thanks for the reply! 
That are good news.
I might come back to you if I have further questions. 
Until then...all the best
